Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Storage/Queries/Merchant/MerchantServiceConfig.hs

```diff
@@ -120,6 +120,7 @@ instance FromTType' BeamMSC.MerchantServiceConfig MerchantServiceConfig where
             createdAt = createdAt
           }
     where
+      -- TODO move to kernel
```

Comment:
I think after converting this to CAC we won't need this function so probably we will remove this function does not make sense also ?